change/@fluentui-react-alert-8a5ecb73-4cf7-422d-9df6-e2c5d0cacffd.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

```suggestion
  "type": "prerelease",
```

This can't be "none" as a release will not be triggered, please set it to "prerelease" (as the package is in beta)

---

@marigome-msft this is still `none`, please update the file 🙏 

```json
{
  "type": "prerelease",
  "comment": "fix: update tokens to match the spec",
  "packageName": "@fluentui/react-alert",
  "email": "marigome@microsoft.com",
  "dependentChangeType": "patch"
}
```

---

yes sorry! I meant to commit the prerelease earlier but somehow it didnt push.